src/md/basic/foundations/deployment.md

```diff
@@ -28,15 +28,15 @@ git push origin master
 Git tags deploy to production:
 
 ```bash
-npm version patch 
+npm version patch
 git push origin master
 ```
 
-That's it!
+Congrats, you've deployed to your production environment!
 
 <h2 id=deploy-arc>Deploy with Architect</h2>
 
-Architect deploys to a `staging` stack by default, will generate and cleanup all the required JSON, YAML and deployment bucket ceremony for you.
+Architect deploys to a `staging` stack by default, and will generate and clean up the required JSON, YAML and deployment bucket for you.
```

Comment:
I wasn't totally sure what this sentence means. Maybe it could be fleshed out? I also thought it might be worth covering setting up an AWS account before the Architect and SAM sections. I wasn't sure how to do that, so I left it as is for now. because you need to set up AWS credentials before deploying using Architect.

Reply:
yeah, think we'll leave the 'setup aws' as a link back to AWS since it is both super complex and always changing